config/inherited_proofing.yml

```diff
@@ -0,0 +1,11 @@
+development:
+  va:
+    base_uri: 'https://staging-api.va.gov'
+
+test:
+  va:
+    base_uri: 'https://staging-api.va.gov'
+
+production:
+  va:
+    base_uri: 'https://api.va.gov'
```

Comment:
So as far as Rails is concerned, our int, staging and prod are all "production", so we use different values in S3 in each environment so that they can point at the right places